app/src/main/java/de/tum/in/tumcampusapp/component/ui/cafeteria/CafeteriaNotificationsProvider.kt

```diff
@@ -0,0 +1,162 @@
+package de.tum.`in`.tumcampusapp.component.ui.cafeteria
+
+import android.app.PendingIntent
+import android.content.Context
+import android.preference.PreferenceManager
+import android.support.v4.app.NotificationCompat
+import android.support.v4.app.NotificationManagerCompat
+import de.tum.`in`.tumcampusapp.R
+import de.tum.`in`.tumcampusapp.component.notifications.NotificationsProvider
+import de.tum.`in`.tumcampusapp.component.notifications.model.AppNotification
+import de.tum.`in`.tumcampusapp.component.notifications.model.FutureNotification
+import de.tum.`in`.tumcampusapp.component.ui.cafeteria.model.CafeteriaWithMenus
+import de.tum.`in`.tumcampusapp.component.ui.cafeteria.model.MenuType
+import de.tum.`in`.tumcampusapp.utils.Const
+import de.tum.`in`.tumcampusapp.utils.DateTimeUtils
+
+class CafeteriaNotificationsProvider(
+        context: Context,
+        private val cafeteria: CafeteriaWithMenus) : NotificationsProvider(context) {
+
+    private val notificationsStore = CafeteriaNotificationsStore(context)
+
+    override fun getNotificationBuilder(): NotificationCompat.Builder {
+        return NotificationCompat.Builder(context, Const.NOTIFICATION_CHANNEL_CAFETERIA)
+                .setAutoCancel(true)
+                .setSmallIcon(R.drawable.ic_cutlery)
+                .setGroup(GROUP_KEY_CAFETERIA)
+                .setGroupSummary(true)
+                .setShowWhen(false)
+                .setColor(notificationColorAccent)
+    }
+
+    private fun getSecondaryNotificationBuilder(): NotificationCompat.Builder {
```

Comment:
please document *why* we need a secondary notificationBuilder here

Reply:
Done.